src/redux/books/books.js

```diff
@@ -0,0 +1,70 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const books = [
+  {
+    id: '1',
+    category: 'Play',
+    title: 'An Enemy of the People',
+    author: 'Dr Stockman',
+    progress: {
+      currentChapter: 'Chapter 17',
+      completed: '52',
+    },
+  },
+  {
+    id: '2',
+    category: 'Novel',
+    title: 'The River Between',
+    author: 'Ngugi Wa Thiongo',
+    progress: {
+      currentChapter: 'Chapter 3: "A Lesson Learned"',
+      completed: '16',
+    },
+  },
+  {
+    id: '3',
+    category: 'Short Story',
+    title: 'The whale Rider',
+    author: 'Suzanne Collins',
+    progress: {
+      currentChapter: 'Module Six',
+      completed: '12',
+    },
+  },
+];
```

Comment:
- [ ] As the previous code reviewer mentioned that the initialState should be an empty array `[]`.